src/components/Pages/ManageWiki/Cards/ConfirmDelete.vue

```diff
@@ -13,30 +13,30 @@
         <v-checkbox
           class="ma-0" hide-details
           label="Was only used for Testing"
```

Comment:
LGTM! Thank you for this change. could you please also change this "T" from capital to small letter